internal/index/indexscanner/scanner_test.go

```diff
@@ -90,6 +95,63 @@ func TestReadPluginFile(t *testing.T) {
 	}
 }
 
+func TestReadReceiptFromFile(t *testing.T) {
+	type args struct {
+		receiptFilePath string
+		pluginName      string
+		indexName       string
```

Comment:
these aren't args.

these are `wantPluginName` `wantIndexName`

You should actually make this `wantStatus` and use cmp.Diff to compare parsed status vs wantStatus.

Reply:
you can see how other table tests do this in other files.